test/functional/operation_changestream_example_tests.js

```diff
@@ -64,6 +67,7 @@ describe('Changestream Examples', function() {
         const changeStream = collection.watch();
         changeStream.on('change', function(change) {
           expect(change).to.exist;
+          changeStream.close();
```

Comment:
sorry for so many of these, but again is this `close` call necessary here if we're already closing the client?

Reply:
yes, to actually close the session and send an `endSession` call, we need to close the change stream.